docs/our_work/ds218_rap_community_of_practice.md

```diff
@@ -1,29 +1,42 @@
 ---
 layout: base
 title: NHS Reproducible Analytical Pipelines (RAP) Champion Squad
-permalink: ds218_rap_community_of_practice.html
+summary: Our roving squad of RAP champions have helped a number of teams not only transform their pipelines, but also taught them how to train others and produced guidance which is used by many other organisations.
+origin: 'NHS Digital'
+tags: ['RAP','Best Practice','Python', 'R']
 ---
 
-> | "Our roving squad of RAP champions have helped a number of teams not only transform their pipelines, but also tought them how to train others and produced guidance which is used by many other organisations."   
-
-<a href = "https://nhsdigital.github.io/rap-community-of-practice/">
-<figure markdown >
-![An image displaying the front page of the NHS RAP Community of Practice website.](../images/ds218_rap_community_of_practice.png) </a>
-<figcaption>Figure 1: The front page of the RAP Community of Practice website.</figcaption>
+<a href="https://www.linkedin.com/feed/hashtag/reproducibleanalyticalpipelines/">
+<figure markdown>
+![Multiple examples of how the team has marketed Reproducible Analytical Pipelines as a way of working, on LinkedIn, Teams, through drop-in sessions, seminars and conference talks.](../images/RAP - marketing examples.png) </a>
+<figcaption>Figure 1: Multiple examples of how the team has marketed Reproducible Analytical Pipelines as a way of working. Clicking on the image will take you through to our feed of Linkedin posts: #ReproducibleAnalyticalPipelines.</figcaption>
```

Comment:
Maybe "Click on the image to see our feed of Linkedin posts: #ReproducibleAnalyticalPipelines"

more of an instruction than a comment - will get more engagement

Reply:
addressed!